packages/recommend-core/src/personalization/getStrategy.ts

```diff
@@ -0,0 +1,34 @@
+type GetStrategy = {
+  appId: string;
+  apiKey: string;
+  region: string;
+};
+
+type StrategyResponse = {
+  facetsScoring: Array<{
+    facetName: string;
+    score: number;
+  }>;
+  personalizationImpact: number;
+};
+
+export const getStrategy = async ({ region, apiKey, appId }: GetStrategy) => {
+  const response = await fetch(
+    `https://recommendation.${region}.algolia.com/1/strategies/personalization`,
```

Comment:
You should be able to get the perso strategy from the [same domain](https://www.algolia.com/doc/rest-api/personalization/#domain) you use to get the affinities (`https://personalization.${region}.algolia.com`).

Reply:
I didn't know that ! Good catch !